docs/index.md

```diff
@@ -0,0 +1,62 @@
+
+# Getting started (napari)
+
+MIDI App Controller is a package designed to integrate MIDI controllers with Python Qt apps using app-model. As of now, it is used most commonly with [napari](napari.org), a viewer for multi-dimensional images. We will show how to use MIDI App Controller with napari but getting started with other applications should look very similar.
+
+## Installation
+
+To install MIDI App Controller in your environment (where Python and napari are already installed), use this command:
+
+(TODO: This doesn't work as the package is not yet published)
+
+```
+pip install midi-app-controller
+```
```

Comment:
I would delete it or comment it out to avoid confusion. I think that the TODO is sufficient.